linux/net/rina/dtcp-utils.c

```diff
@@ -346,6 +349,14 @@ static struct dtcp_config * dtcp_config_create_gfp(gfp_t flags)
           }
         */
 
+        tmp->dtcp_ps = policy_create();
```

Comment:
Ok for this specific case, but.. what about all the other cases? For example tx_control field in struct window_fctrl_config? It seems completely useless (waste priceless kernel memory and complicates code). I know, it's a separate issue from this Pull Request.